x-pack/plugins/observability_solution/inventory/public/components/app_root/index.tsx

```diff
@@ -43,12 +43,12 @@ export function AppRoot({
   return (
     <InventoryContextProvider context={context}>
       <RedirectAppLinks coreStart={coreStart}>
-        <InventorySearchBarContextProvider>
-          <RouterProvider history={history} router={inventoryRouter}>
+        <RouterProvider history={history} router={inventoryRouter as any}>
```

Comment:
Why `inventoryRouter as any`?

Reply:
It was throwing a warning, but I fixed it now.

Reply:
I thought I had fixed it but it actually broke all path types on the components. So reverting it to any... we also do it on APM and other plugins, so won't be looking into it for now.